concordium-node/src/grpc2.rs

```diff
@@ -1086,7 +1086,22 @@ pub mod server {
                     }
                 });
                 let service = service::queries_server::QueriesServer::new(server);
-                let log_layer = tower_http::trace::TraceLayer::new_for_grpc();
+                let log_layer = {
+                    use tower_http::classify::GrpcCode as Code;
+                    // Change the trace level for gRPC status codes
+                    let classifier = tower_http::classify::GrpcErrorsAsFailures::new()
+                        .with_success(Code::Cancelled)
+                        .with_success(Code::InvalidArgument)
+                        .with_success(Code::NotFound)
+                        .with_success(Code::AlreadyExists)
```

Comment:
RESOURCE_EXHAUSTED can be argued both places. We actually have user limits right? So I would say that maybe also add that. Could that perhaps be made something like a warning? (Could also indicate a try on a DoS attack)